docs/enhancements/metallb-operator.md

```diff
@@ -0,0 +1,205 @@
+
+Unlike for the **internal** network, this enhancement proposes to always infer the subnet of the
+external network from the External API VIP and the External Ingress IP provided by the user. There
+are utilities in Assisted Installer to do this already.
+
+Some requirements:
+
+- The **external** API VIP and Ingress IP must be on the same subnet.
+- The host must be connected to this network and the network must be routable
+- These 2 IPs must be optional. The IPAddressPool and Service won't be configured if this information is not provided
+- Either both are IPs are provided or none.
+- The *internal* network must be configured as a static network to prevent the MachineNetwork to
+change and, therefore, the IP of each of the nodes.
+
+### User Stories
+
+#### Story 1
+
+As a hardware provider, I want to be able to deliver an OCP appliance that can be shipped to a
+different location and reconnected.
+
+#### Story 2
+
+As a datacenter administrator, I want to be able to install an OCP cluster whose IP can be
+reconfigured as well as its domain.
+
+### Implementation Details/Notes/Constraints [optional]
+
+1. It's currently not possible to enable operators when using KubeAPI. Implementing this feature,
+without enabling it through KubeAPI will limit the scope of it for environments that use ZTP. It
+should be considered to allow enabling operators through KubeAPI either as part of this enhancement
+or a separate one.
+2. Assisted Installer doesn't have a way to configure an operator. Operators can be enabled or
+disabled. The rest of the information they take from the infrastructure data that has been
+collected. For this enhancement, we need to either introduce the concept of External API VIP and
+External Ingress IP to the cluster or allow for specifying operator-specific configurations.
+
+### Risks and Mitigations
+
+Previous sections of the enhancement state that the MachineNetwork must be static for this workflow
+to work as expected. It is important that this network won't change during a relocation or at any
+point during the lifecycle of the cluster.
+
+This requirement raises the risk of this enhancement as it introduces new infrastructure
+requirements during the deployment and at the new location in case of relocation. Specifically, in
+the case of a customer using a network accessible to more than one cluster as their MachineNetwork,
+there's a risk of having IP clashes.
+
+People using ZTPFW have solved this issue by providing an isolated, routable, network as the
+MachineNetwork. This is done either by using a top of the rack switch or some other mechanism. This
+is particularly relevant for multinode deployments where hosts need to be able to talk to each
+other.
+
+Finally, the UX won't be as straightforward when building a relocatable cluster as users will have
+to be familiar with the 2 networks that are needed.
+
+## Design Details [optional]
+
+### Open Questions
+
+### UI Impact
```

Comment:
also to d/s the unboxing UI? and include it here?

Reply:
mmmh, I like this idea. If we are building a relocatable cluster, we might as well ship the UI that allows to reconfigure things. But, at this point, I wonder if it should/could be an extension on the actual OpenShift console rather than a completely independent container/ui. IIRC, we have a way to do this @jelkosz ?